src/components/dashboard/components/dictionary/DictionaryContainer.jsx

```diff
@@ -73,7 +73,7 @@ export class DictionaryOverview extends Component {
     } = this.props;
     const versionUrl = `/${ownerType}/${owner}/${type}/${name}/versions/?verbose=true`;
 
-    if (prevProps.isReleased !== this.props.isReleased) {
```

Comment:
What has this got to do with the invalid date?

Reply:
Here, the action `fetchVersions` (which fetches all the details of released versions - including the release date) needs to be triggered whenever a new version is released. However, the property `isReleased` changes only once - when the first version is released. Since we need to call `fetchVersions` on every new release, monitoring a change in this won't work because its value will only change on the first release. 

The number of versions released, however, increases with every release. I, therefore, changed the logic to be monitoring this and call `fetchVersions` whenever there is a change - which would occur every time a new collection version is released.

Reply:
Is that what causes the display of an invalid date?

Reply:
Yes,  because the value of the date would be null - which can't be formated hence 'invalid date' would be returned.